example/package.json

```diff
@@ -24,7 +24,6 @@
     "load-test": "../node_modules/.bin/jasmine ./spec/loadTest/runScaledtest.js",
     "package": "for x in lambdas/*; do (echo \"packaging $x\" && cd $x && test -e package.json && npm run package); done",
     "parallel-tests": "sh scripts/tests-parallel.sh",
-    "python-lint": "for x in lambdas/*; do cd $x && npm run python-lint && cd -; done",
```

Comment:
Was this removed because it isn't called by the root `package.json` -> `lint` script? If so, great clean-up!